src/coresyftools/coresyftools/tool.py

```diff
@@ -38,9 +59,27 @@ def __init__(self, run_script_file_name):
             self.context_directory, self.MANIFEST_FILE_NAME)
         self.manifest = get_manifest(self.manifest_file_name)
         self.arg_parser = CoReSyFArgumentParser(self.manifest)
+        if 'command' in self.manifest:
+            self._validate_command(self.manifest['command'])
         self.operation = self.manifest.get('operation', {})
         self._validate_operation(self.operation)
 
+    def _validate_command(self, command):
+        placeholders = self._extract_command_placeholders(command)
+        for placeholder in placeholders:
+            if placeholder not in self.arg_parser.identifiers:
+                raise UnexpectedCommandPlaceholder(placeholder)
+        for identifier in self.arg_parser.identifiers:
+            if identifier not in placeholders:
+                raise MissingCommandPlaceholderForOption(identifier)
+
+    def _extract_command_placeholders(self, command_template):
+        formatter = string.Formatter()
+        return set([field_name for
+                    literal_text, field_name, format_spec, conversion in
+                    formatter.parse(command_template)])
+
+
     def _validate_operation(self, operation_dict):
```

Comment:
This method is as confusing as the `_validate_manifest` in `argument_parser.py`. What does this do? Why does it exist? It has a declared parameter `operation_dict`, which isn't used, and returns some hardcoded stuff. If it's expected, please comment the method so that it is clear why this exists.

Reply:
Added comment.